internal/provider/provider.go

```diff
@@ -160,6 +162,8 @@ func (p *BloxOneProvider) Resources(_ context.Context) []func() resource.Resourc
 		ipamfederation.NewFederatedBlockResource,
 
 		redirect.NewCustomRedirectResource,
+
+		clouddiscovery.NewProvidersResource,
```

Comment:
The naming convention is usually singular for resource and plural for data source

Reply:
Fixed